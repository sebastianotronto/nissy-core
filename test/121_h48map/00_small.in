11
7
4
34
12
45
7
34
13
45
5
