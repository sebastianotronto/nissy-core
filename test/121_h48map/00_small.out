2
34 12
45 5
